boards/nxp/frdm_mcxn236/doc/index.rst

```diff
@@ -65,6 +65,8 @@ The FRDM-MCXN236 board configuration supports the following hardware features:
 +-----------+------------+-------------------------------------+
 | ADC       | on-chip    | adc                                 |
 +-----------+------------+-------------------------------------+
+| USBHS     | on-chip    | USB device                          |
++-----------+------------+-------------------------------------+
```

Comment:
Hi @danieldegrasse , @dleach02 , USB feature already be added in doc index.rst file.